front/src/applications/stdcm/components/StdcmResults/StdcmResults.tsx

```diff
@@ -67,9 +68,12 @@ const StcdmResults = ({
     );
   }, [outputs]);
 
-  const simulationPathSteps = hasSimulationResults
-    ? outputs.results.simulationPathSteps
-    : undefined;
+  const simulationPathSteps = useMemo(() => {
+    if (!hasSimulationResults) {
+      return undefined;
+    }
+    return extractMarkersInfo(outputs.results.simulationPathSteps);
+  }, []);
```

Comment:
You probably should add `hasSimulationResults` and `outputs` in the deps

Reply:
You're right, thanks !